### ISSUE TITLE
Can't use the app where YouTube Music is not available

### ISSUE THREAD

**Issue by th3-s7r4ng3r** [vfsfitvnm/ViMusic#999]
### Steps to reproduce the bug

As Youtube Music is not available in my country, I can't use this awesome app. Doesn't show up anything, Not even after searching for a song. Could use a VPN to overcome that, but it's not ideal. If possible add a source from another country even when YouTube Music is not available.

### Expected behavior

Play music and display information.

### Actual behavior

Nothing shows up. Every category is empty.

### ViMusic version

0.5.4

### Android version

Android 13

**Comment by Redskull-127** [comment]
hey @th3-s7r4ng3r, did connecting to VPN or proxy worked?
if yes let me know asap.

**Comment by th3-s7r4ng3r** [comment]
Yes, I tried a VPN, then the app works fine. Could you please add something to use a different source for the app if YT Music not available for that country? That would definitely increase the user base more.

**Comment by Redskull-127** [comment]
Cool, started working on it.
Btw till then use google DNS to bypass this.

8.8.8.8
8.8.4.4

**Comment by th3-s7r4ng3r** [comment]
Didn't know, I'll give it a try. And waiting for the new release :)
